auto_install/install.sh

```diff
@@ -296,6 +311,36 @@ distroCheck(){
 				;;
 			esac
 		;;
+		Alpine)
+		PKG_MANAGER='apk'
+		UPDATE_PKG_CACHE="${PKG_MANAGER} update"
+		PKG_INSTALL="${PKG_MANAGER} --no-cache add"
+		PKG_COUNT="${PKG_MANAGER} --no-cache upgrade -s | grep -sEe ' [0-9]+ packages' | sed -E -e 's/.* ([0-9]+) packages/\1/' || true"
+		CHECK_PKG_INSTALLED="${PKG_MANAGER} --no-cache info -e"
+
+		# install grepcidr manually
+		## install dependencies
+		"${PKG_INSTALL}" build-base make curl tar
```

Comment:
```
+ 'apk --no-cache add' build-base make curl tar
./install.sh: line 324: apk --no-cache add: command not found
```

`"${PKG_INSTALL}"` shouldn't be quoted since you actually need word splitting.